docs/testing-flows.md

```diff
@@ -59,9 +59,13 @@ These flows can be used when testing PRs or releases, and as a basis for documen
   - Private, draft or trashed products.
   - Products with price = 0.
   - Products out of stock if the option `woocommerce_hide_out_of_stock_items` is enabled (since `v1.0.9`).
-  - Grouped products.
-  - Parent product of Variable products (since `v1.0.9`). The children variations are included.
-- The product types excluded from the feed can be modified by the `pinterest_for_woocommerce_excluded_product_types` filter.
+- The plugin supports by default the following product types (since `v1.2.9`):
```

Comment:
```suggestion
- The plugin product feed includes the following product types by default (since `v1.2.9`):
```

Reply:
@alexiglesias31 , please review the suggestion, I think it would sound a bit better. What do you think?